core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -35,12 +35,23 @@
    */
   public static <T> T call(Logger logger, RpcCallable<T> callable) throws AlluxioTException {
     try {
-      return callable.call();
+      logger.debug("Enter: {}", callable);
+      T ret = callable.call();
+      logger.debug("Exit (OK): {}", callable);
+      return ret;
     } catch (AlluxioException e) {
-      logger.debug("{}, Error={}", callable, e.getMessage());
+      if (logger.isDebugEnabled()) {
+        logger.debug("Exit (Error): {}", callable, e);
+      } else {
+        logger.warn("{}, Error={}", callable, e.getMessage());
+      }
       throw AlluxioStatusException.fromAlluxioException(e).toThrift();
     } catch (RuntimeException e) {
-      logger.error("{}", callable, e);
+      if (logger.isDebugEnabled()) {
+        logger.debug("Exit (Error): {}", callable, e);
+      } else {
+        logger.error("{}", callable, e);
+      }
```

Comment:
simplify to `logger.error("Exit (Error): {}", callable, e)`?

Reply:
done